src/pygeom.c

```diff
@@ -114,9 +118,11 @@ static PyObject* GeometryObject_ToWKT(GeometryObject* obj) {
   }
 
   GEOSWKTWriter_setRoundingPrecision_r(ctx, writer, precision);
+#if !GEOS_SINCE_3_12_0
   GEOSWKTWriter_setTrim_r(ctx, writer, trim);
   GEOSWKTWriter_setOutputDimension_r(ctx, writer, dimension);
```

Comment:
Those have the correct defaults or are no longer needed for GEOS 3.12? Can you add a brief comment explaining this?